File: UnicoVehicle/UnicoVehicle.DTO/TestDrive.cs
using System;
namespace UnicoVehicle.DTO
{
    public class TestDrive
    {
        public int TestDriveId { get; set; }
        public Miscellaneous.User User { get; set; }
        public Miscellaneous.Showroom Showroom { get; set; }
        public Miscellaneous.Vehicle Vehicle { get; set; }
        public Status TestDriveStatus { get; set; }
        public DateTime TestDriveDate { get; set; }
    }
}

File: Flutter App/unicoVehicle/lib/Models/TestDrive.dart
import './Status.dart';
import './Miscellaneous/User.dart';
import './Miscellaneous/Vehicle.dart';
import './Miscellaneous/Showroom.dart';

class TestDrive {
  final int testDriveId;
  final User user;
  final Vehicle vehicle;
  final Showroom showroom;
  final Status status;
  final DateTime testDriveDateTime;

  TestDrive({
    this.showroom,
    this.status,
    this.testDriveDateTime,
    this.testDriveId,
    this.user,
    this.vehicle,
  });
}
